docs/mllib-data-types.md

```diff
@@ -431,7 +454,42 @@ long n = mat.numCols();
 // Drop its row indices.
 RowMatrix rowMat = mat.toRowMatrix();
 {% endhighlight %}
-</div></div>
+</div>
+
+<div data-lang="python" markdown="1">
+
+An [`IndexedRowMatrix`](api/python/pyspark.mllib.html#pyspark.mllib.linalg.IndexedRowMatrix) can be created from an 
+`RDD` of IndexedRows, where [`IndexedRow`](api/python/pyspark.mllib.html#pyspark.mllib.linalg.IndexedRow) is a 
+wrapper over `(long, Vector)`.  An `IndexedRowMatrix` can be converted to a `RowMatrix` by dropping its row indices.
```

Comment:
Yeah, although it doesn't matter for Python 3, Python 2 still has the distinction, and regardless, from an API/documentation perspective, I think it's a good idea to mimic the requirements of the Scala side, which uses a long in that case.